eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/rest/ClusteredVariantsRestController.java

```diff
@@ -187,17 +189,23 @@ public ClusteredVariantsRestController(
         List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>> clusteredVariants =
                 new ArrayList<>();
 
-        Optional<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>> clusteredVariant =
+        Optional<AccessionWrapper<IClusteredVariant, String, Long>> clusteredVariantWrapper =
                 nonHumanActiveService.getByIdFields(assembly, chromosome, start, variantType);
-        clusteredVariant.ifPresent(clusteredVariants::add);
+        clusteredVariantWrapper.ifPresent(wrapper -> clusteredVariants.add(toDto(wrapper)));
 
-        Optional<List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>>> humanClusteredVariant =
+        Optional<List<AccessionWrapper<IClusteredVariant, String, Long>>> humanClusteredVariants =
                 humanService.getByIdFields(assembly, chromosome, start, variantType);
-        humanClusteredVariant.ifPresent(clusteredVariants::addAll);
+        humanClusteredVariants.ifPresent(wrappers -> clusteredVariants.addAll(
+                wrappers.stream().map(this::toDto).collect(Collectors.toList())));
 
         return clusteredVariants.isEmpty() ? ResponseEntity.notFound().build() : ResponseEntity.ok(clusteredVariants);
     }
 
+    private AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long> toDto(
```

Comment:
Minor quibble here... Is toDTO acceptable when it comes to camel case? that seems more readable... I almost read this as todo...

Reply:
I would vote for `toDto` for consistency in acronyms across EVA java code, but given that in accession commons they used `AccessionResponseDTO` (used here), I am ok prioritising local consistency over global consistency.